### PR TITLE
[ja] Localize Zero Trust Architecture for Japanese

### LINKED ISSUE #2762: [ja] Localize Zero Trust Architecture for Japanese
Localize https://glossary.cncf.io/zero-trust-architecture/

### PR DESCRIPTION
**Pull request by Okabe-Junya**
### Describe your changes

Localize Zero Trust Architecture for Japanese

### Related issue number or link (ex: `resolves #issue-number`)

resolves #2762 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by naonishijima — APPROVED** [review]
LGTM

File: content/ja/zero-trust-architecture.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+信頼があるため問題がないという前提があります。
+しかし、ゼロトラストアーキテクチャは、その信頼が脆弱性になると認識しています。
+攻撃者が信頼されたデバイスへのアクセスを獲得した場合、
+そのデバイスに与えられている信頼のレベルとアクセスに依存してシステムは攻撃に対して脆弱になります。
+なぜなら、攻撃者は信頼されたネットワークの境界内におり、システム内を横断的に移動することができるからです。
+ゼロトラストアーキテクチャでは、信頼が取り除かれるため攻撃者はシステム内をさらに横断する前に検証を強いられ、結果として攻撃される場所が減ります。
+
+## どのように役に立つのでしょうか
+
+ゼロトラストアーキテクチャを採用することで、主な利点としてセキュリティが向上し攻撃される場所が減少します。
```
Comment (kaitoii11) [inline content/ja/zero-trust-architecture.md:27]: attack surfaceはアタックサーフィスか攻撃対象領域と訳しちゃっていいと思います

Comment (Okabe-Junya) [reply content/ja/zero-trust-architecture.md]: ありがとうございます。

「アタックサーフェス」という用語を知らなかったです。。https://github.com/cncf/glossary/pull/2764/commits/11a4104e0e020bdd504b6c59596d53472b5b1d6e https://github.com/cncf/glossary/pull/2764/commits/e9668603c4f24f4e9c06ffe63accacb09c52407e で修正しました。

reduceの訳は、「減少」と「縮小」のどちらの方が良いでしょうか？？（個人的には縮小の方が適切ではないかと思っています）

Comment (Okabe-Junya) [reply content/ja/zero-trust-architecture.md]: https://github.com/cncf/glossary/pull/2764/commits/ae0d78f7c6d1ea383435ea41c256b3496bbb62e3 で修正しました🙆‍♂️

**Review by kaitoii11 — APPROVED** [review]
LGTM

**Pull request merged**